### DISCUSSION TITLE
Query regarding when parsedSize and gzipSize are calculated and shown

### DISCUSSION THREAD

**Discussion by Pirororor** [webpack-contrib/webpack-bundle-analyzer#648] [Q&A]
When inspecting the json of the report file, I see that there are quite a few components that have the following

1. assetSize, gzipSize non-zero, but "inaccurateSizes" is "true"
2. "parsedSize: 0" or "gzipSize: 0" 
![image](https://github.com/user-attachments/assets/2e6ac895-ba0d-4a6a-9da8-b095df75b428)
4. only assetSize field exists

Can I check under what circumstances each scenario is shown? From what I know, (1) is shown for concatenated modules, but I am not sure for (2) and (3)

**Comment by Pirororor** [comment]
anyone..?

**Comment by valscion** [comment]
Good question! I don't unfortunately have an answer off the top of my head right now.

Is there some commonality between the groups that have the sizes as 0?

I haven't dug too deep to the JSON representation of the UI graph myself so can't really help that much in there. You might need to figure this out a bit on your own. Feel free to post any insights or suggestions for the cause you could come up with, maybe someone will be able to help you ☺️
